tests/integration/components/osf-link/component-test.ts

```diff
@@ -0,0 +1,143 @@
+import { render} from '@ember/test-helpers';
+import Ember from 'ember';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+
+module('Integration | Component | osf-link', hooks => {
+    setupRenderingTest(hooks);
+
+    let orgOnError: any;
+    hooks.beforeEach(function() {
+        orgOnError = Ember.onerror;
+    });
+    hooks.afterEach(function() {
+        Ember.onerror = orgOnError;
+    });
+
+    test('it renders the osf-link correctly for an @href', async assert => {
+        await render(hbs`
+            <OsfLink
+                data-test-get-started-button
+                data-analytics-name='Get started button'
+                class='btn btn-primary'
+                local-class='startButton'
+                @target='_blank'
+                @href='https://osf.io'
+            >
+                {{t 'dashboard.getting_started'}}
+            </OsfLink>
+        `);
+        assert.dom('[data-test-get-started-button]').exists();
+        assert.dom('[data-test-get-started-button]').hasText('Getting Started');
+        assert.dom('[data-test-get-started-button]').hasClass('btn');
+        assert.dom('[data-test-get-started-button]').hasClass('btn-primary');
+        /* eslint-disable max-len */
+        assert.dom('[data-test-get-started-button][href$="https://osf.io"]').exists('The href https://osf.io is not in the dom');
```

Comment:
We can just split this assert into two lines and avoid the `eslint-disable`

```suggestion
        assert.dom('[data-test-get-started-button][href$="https://osf.io"]')
            .exists('The href https://osf.io is not in the dom');
```